Jenkinsfile

```diff
@@ -66,7 +66,7 @@ node(platform) {
 
     stage("Archive") {
         if (platform == 'android') {
-            archiveArtifacts artifacts: 'platforms/android/build/outputs/apk/android-${BUILD_CONFIG}.apk', excludes: 'platforms/android/build/outputs/apk/*-unaligned.apk'
```

Comment:
it's ok as it does not use any variables, we need to use double quotes to parse a variable(s) value(s).